content/docs/sidebar.json

```
@@ -102,7 +102,14 @@
         "label": "How To",
         "slug": "how-to",
         "source": false,
-        "children": ["undo-adding-data", "update-tracked-files"]
+        "children": [
+          {
+            "label": "Add Output to Stage",
+            "slug": "add-output-to-stage"
+          },
```

Quick Q @rogermparent. Do you think it would be easy to NOT capitalize certain words from slugs like "to"? To avoid having to use labels for instances like this one. Other such words are 'a' and 'the'. We could keep a list somewhere. Just checking with you before even opening a feature request, may not be worth it.

---

I believe you are describing [Title Case](https://en.wikipedia.org/wiki/Title_case), and if you are it should be pretty easy to use [this small library](https://www.npmjs.com/package/title-case) in our sidebar label generation.

Let me refamiliarize myself with the code to make sure this works, but I'm pretty confident it's doable.

---

While it seems using an external library can cause issues with words specific to us, like "DVC", we can simply write a small function that does automatic casing the specific way we want. An external file for this is be a bit much in my opinion, but a small custom function would be a clean solution that can adapt to any of our needs.

---

Yeah, title capitalization, basically. I think if any word is all caps in the slug it should let it be, but if the slug uses lower case we can just use the a sidebar.json label for titles with "DVC".

---

I made #1844 as a quick answer to this. It does title case and capitalizes "DVC" I figure we want it capitalized more often than not, and the behavior can be overridden with a `label`.

---

@imhardikj please merge master and apply this, double check the label looks good. Thanks:
```suggestion
          "add-output-to-stage"
```